docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -103,6 +103,17 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 <3> You define a resource pool for the off-heap.
 <4> You define a resource pool for the disk.
 
+==== Byte Sized Tiering
+
+[source,java,indent=0]
+----
+include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag=byteSizedTieredCache]
+----
+
+<1> You can also size a tiered cache based on bytes only.You can limit the amount of java heap used by cache for storing key-value pairs. 
```

Comment:
Would rephrase that:
You can also size the heap tier in bytes. This will limit the amount of heap used by that tier for storing key-value pairs. Note that there is a cost associated to sizing objects.